.github/workflows/diarizer-benchmark.yml

```
@@ -1,4 +1,4 @@
-name: Diarizer Performance Benchmark
+name: Streaming Diarization Benchmark
```

```suggestion
name: Diarizer Performance Benchmark
```

lets not change this